include/nut_float.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * nut_float.h - Network UPS Tools include files for floating point routines
+ * and ways to compare if non-integers are close enough to assume equal
+ *
+ * Copyright (C) 2011	Arjen de Korte <adkorte-guest@alioth.debian.org>
```

Comment:
Is this from one of Arjen's other files? If not, an updated copyright year and author might be good.

Reply:
Good point, did not commit the change. This file is new :)